gradoop-flink/src/main/java/org/gradoop/flink/io/impl/tlf/functions/TLFFileFormat.java

```diff
@@ -57,23 +58,26 @@
    */
   @Override
   public String format(GraphTransaction graphTransaction) {
+    StringBuilder builder = new StringBuilder();
+
     Map<GradoopId, Integer> vertexIdMap = Maps
       .newHashMapWithExpectedSize(graphTransaction.getVertices().size());
 
-    Collection<String> lines = Lists.newArrayListWithExpectedSize(
-      graphTransaction.getVertices().size() +
-        graphTransaction.getEdges().size() + 1
-    );
-
     // GRAPH HEAD
-    lines.add(TLFGraph.SYMBOL + " # " + graphId);
+    builder.append(String.format("%s %s %s%n",
```

Comment:
add a short documentation on how a line would look like, see [EdgeToJson](https://github.com/dbs-leipzig/gradoop/blob/c7b67d851288901ed350a737021a7624e4989ef4/gradoop-flink/src/main/java/org/gradoop/flink/io/impl/json/functions/EdgeToJSON.java) for an example